data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -384,7 +386,7 @@
         {VARIABLE_OP malin.experience add 8}
         [floating_text]
             x,y=$malin.x,$malin.y
-            text="<span color='#800080'>" + "+8 exp" + "</span>"
+            text=_ "<span color='#800080'>" + "+8 exp" + "</span>"
```

Comment:
@nemaara looks like we should put an underscore on this line, but applying to the "+8 exp" string rather than the color strings.

Might just make the whole line one translatable string, as we already throw marked-up text at the translators.

Reply:
I think the underscore should indeed go after the first `+`. Throwing markup text at translators is perhaps okay for complex text such as the help document, but I don't think it should be done in avoidable situations like this.

Reply:
No idea what the right way is but sure making it translatable is totally fine.